backend/serve.go

```diff
@@ -85,7 +84,6 @@ func GRPCServeOpts(opts ServeOpts) grpcplugin.ServeOpts {
 //   - otel grpc stats handler (see otelgrpc.NewServerHandler)
 func grpcServerOptions(serveOpts ServeOpts, customOpts ...grpc.ServerOption) []grpc.ServerOption {
 	options := defaultGRPCMiddlewares(serveOpts)
-	options = append(options, grpc.StatsHandler(otelgrpc.NewServerHandler()))
```

Comment:
@xnyo asking you since I'm guessing you might have been involved here. I moved this into defaultGRPCMiddlewares and doesn't seem to harm to me?

Reply:
Yep, lgtm :+1: 

I have tested it and it still works fine